File: app/src/main/java/com/example/weizhenbin/wangebug/image/ImageLoadListenerAdapter.java
package com.example.weizhenbin.wangebug.image;

import android.graphics.Bitmap;
import android.graphics.drawable.Drawable;

/**
 * Created by weizhenbin on 2018/8/15.
 */

public class ImageLoadListenerAdapter implements IImageLoadListener{


    @Override
    public void onLoadStart(String url, Drawable defDrawable) {

    }

    @Override
    public void onLoadError(String url, Drawable errorDrawable) {

    }

    @Override
    public void onLoadSuccess(Bitmap bitmap, String url) {

    }
}

File: app/src/main/java/com/example/weizhenbin/wangebug/image/ImageLoadListenerAdapter.kt
package com.example.weizhenbin.wangebug.image

import android.graphics.Bitmap
import android.graphics.drawable.Drawable

/**
 * Created by weizhenbin on 2018/8/15.
 */

open class ImageLoadListenerAdapter : IImageLoadListener {


    override fun onLoadStart(url: String, defDrawable: Drawable?) {

    }

    override fun onLoadError(url: String, errorDrawable: Drawable?) {

    }

    override fun onLoadSuccess(bitmap: Bitmap, url: String) {

    }
}
